<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BqWidget</class>
 <widget class="QWidget" name="BqWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>摄像头联动</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 255, 127);</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>601</width>
              <height>331</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <widget class="QWidget" name="gridLayoutWidget">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>581</width>
                  <height>311</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="2" column="0" colspan="2">
                  <widget class="QWidget" name="widget_5" native="true"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QListView" name="listView"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QWidget" name="widget_3" native="true"/>
                 </item>
                 <item row="1" column="2">
                  <widget class="QWidget" name="widget_4" native="true"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QWidget" name="widget_6" native="true"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>摄像头管理</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>351</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>601</width>
              <height>341</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QWidget" name="widget_8" native="true"/>
             </item>
             <item>
              <widget class="QTableView" name="tableView"/>
             </item>
             <item>
              <widget class="QWidget" name="widget_9" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
